lib/recurly/util/xml_tools.php

```diff
@@ -105,7 +105,9 @@ class XmlTools
   // @returns DOMDocument
   public static function createDocument($version = '1.0')
   {
-    return new DOMDocument($version);
+    $doc = new DOMDocument($version);
+    $doc->encoding = 'utf-8';
+    return $doc;
```

Comment:
Seems like we could use this method in all the above files that are creating `DOMDocument`s? That was the idea of having this thing be a static class, after all. 👍